exercises/accumulate/canonical-data.json

```diff
@@ -23,43 +23,85 @@
       "description": "accumulate squares",
       "property": "accumulate",
       "input": {
-        "list": [1, 2, 3],
+        "list": [
```

Comment:
These don't seem desirable. They make reading and copy and pasting cases harder when manually implementing a solution.